1 Foundations/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+    </head>
+    <body>
+        <h1 id="date"></h1>
+        <h1 id="hello"></h1>
```

Comment:
No HTML Files needed in this repo! Read the base readme. The commits must be pure JS code. Explain how to run them in notes.

Reply:
Oh, I thought it would be better. I will remove it